src/components/icons/index.ts

```diff
@@ -10,8 +10,10 @@ export { ChevronLeftSmallIcon } from './ChevronLeftSmallIcon';
 export { ChevronRightSmallIcon } from './ChevronRightSmallIcon';
 export { ChevronRightTinyIcon } from './ChevronRightTinyIcon';
 export { ErrorIcon } from './ErrorIcon';
+export { CallCtaIcon } from './CallCtaIcon';
```

Comment:
I would suggest to remove the `Cta`. This says what it should be used for and not what it represents. I could use this icon for other things.
`CallIcon` and `MessageIcon`